app/templates/eventView.html

```diff
@@ -66,7 +67,7 @@ <h5 id="location"style="margin: 0;">Location:</h5>
                     <br>
                     <div>
                         {% if eventData.isRsvpRequired %}
-                            {% if not eventPast %}
+                            {% if eventPast != event.isPastEnd %}
```

Comment:
Seems like the current logic only shows the RSVP stuff if the event is in progress (when isPastStart isn't equal to isPastEnd)

Reply:
`if not event.isPastStart`